package.json

```diff
@@ -63,94 +63,94 @@
     "arrowParens": "always"
   },
   "devDependencies": {
-    "@babel/cli": "^7.7.7",
-    "@babel/core": "^7.7.7",
-    "@babel/node": "^7.7.7",
-    "@babel/plugin-proposal-class-properties": "^7.7.4",
-    "@babel/plugin-proposal-object-rest-spread": "^7.7.7",
-    "@babel/preset-env": "^7.7.7",
-    "@babel/preset-flow": "^7.7.4",
-    "@babel/preset-react": "^7.7.4",
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.10.5",
+    "@babel/node": "^7.10.5",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
+    "@babel/preset-flow": "^7.10.4",
+    "@babel/preset-react": "^7.10.4",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.0.3",
+    "babel-eslint": "^10.1.0",
     "babel-jest": "^24.9.0",
-    "babel-plugin-inline-react-svg": "^1.1.0",
-    "babel-plugin-source-map-support": "^2.1.1",
+    "babel-plugin-inline-react-svg": "^1.1.1",
+    "babel-plugin-source-map-support": "^2.1.2",
     "commander": "^2.20.3",
     "cpx": "^1.5.0",
-    "electron": "^7.1.9",
+    "electron": "^7.3.2",
```

Comment:
This is a super-generic question, but what are the reasons to stay on Electron 7, rather than move to 8 or 9? Are there upward compatibility problems with the move?

Reply:
Nope, I don't have any reason to stay on Electron 7. I just looked through the electron releases pages and didn't find any obvious breaking changes that would effect our app. I'll upgrade electron to the latest version in this PR.